docs/pages/blog/mui-x-v6-alpha-zero.md

```diff
@@ -0,0 +1,97 @@
+---
+title: A major update is coming for the advanced components.
```

Comment:
Sorry I dropped the ball on Friday! I like the direction you're going in with this title. What if we added a hook to address the reader and hint at our call to action?

```suggestion
title: A major update is coming for MUI X—and you can get involved
```